package.json

```diff
@@ -28,7 +28,7 @@
     "joi": "^8.1.0",
     "lodash": "^4.12.0",
     "logr": "^1.8.0",
-    "rapptor": "firstandthird/rapptor#13332ac"
+    "rapptor": "firstandthird/rapptor#e1b04d6d8dd"
```

Comment:
Seemed to work for me? The `sha` is actually: `e1b04d6d8dd09d81ec6ea3111a451b97d7784332`